CHANGES.md

```diff
@@ -8,6 +8,7 @@
 - #1199 Can't get back from to login screen from registration screen
 - #1214 Setting `allow_contact_requests` to `false` has no effect
 - #1222 Adding a bookmark should prefill the room name
+- #1209 Bring color codes of users' status in line with other clients
 
```

Comment:
Please make sure these are sorted according to issue number.

Reply:
Oh I didn't realise it was sorted that way. Thanks, will change.